### PR TITLE
Add package.json file

### PR DESCRIPTION
**Pull request by goldenapples**
This repo includes a grunt uglify task to build the dist files, but not the package.json required to install grunt. This adds a basic package.json which installs the local modules for `grunt` and `grunt-contrib-uglify` required for developing on the plugin.

### CODE REVIEW

**Comment by goldenapples** [comment]
Oh, I didn't see the instructions from #46 when I first opened this PR. If it makes more sense not to include the package.json file, as the CONTRIBUTING document from that PR suggests, then feel free to close this PR out and use the instructions there.

**Comment by roborourke** [comment]
Yeah I'm not sure why we don't include that to be honest. Will simplify development and match expectations more closely. I'm in favour.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "gaussholder",
+  "version": "1.1.3",
+  "description": "Quick and beautiful image placeholders using Gaussian blur.",
+  "main": "Gruntfile.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (roborourke) [inline package.json]: Can you add a build script? And perhaps update the `readme.md`? Using an `npm` script will ensure the local version of grunt is used.

Comment (goldenapples) [reply package.json]: I've added a build script in 50e448b. I'm thinking info about the build script should probably go in the CONTRIBUTING.md file from #46, as the readme doesn't have an development-specific information. Or were you talking about some other updates to the readme?

Comment (roborourke) [reply package.json]: Oh, no it's cool I'll update my PR to that effect

**Comment by roborourke** [comment]
Actually I realised the circleci config could be updated based on this too. I can handle that.

**Pull request merged**